scripts/parse-wpsummaries.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import fileinput
+import json
+import spacy
+
+from readysetdata import wikipedia, parse_jsonl
+
+nlp = spacy.load("en_core_web_sm")
+
+for row in parse_jsonl(fileinput.input()):
+    if row.title.startswith("List of "):
+        # Exclude "List" articles
+        continue
+
+    text = row.revision.text['#text']
+    d = wikipedia.get_first_paragraph(text)
+    out = dict(wp_title=row.title, shard=row.title[0].lower())
+    out.update(d)
+
+    if "first_paragraph" in out:
+        sentences = list(nlp(out["first_paragraph"]).sents)
+        if sentences:
+            out["first_sentence"] = str(sentences[0])
+        else:
+            out["first_sentence"] = out["first_paragraph"]
```

Comment:
I think we should probably functionize this and call it from the existing pipeline, so we're not downloading/processing the entire corpus twice.  (We are planning to run all the code in readysetdata ourselves once a month or so, so it might make a material difference :)  And then we wouldn't need wikisummaries.sh, though we would need to figure how we'll split the stream in the other shell script.